lxd-agent/devlxd.go

```diff
@@ -258,7 +258,7 @@ func socketUnixRemoveStale(path string) error {
 	return nil
 }
 
-// Change the file mode of the given unix socket file,
+// Change the file mode of the given unix socket file,.
```

Comment:
The comma at the end needs to be removed.

Reply:
Done.